jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -249,19 +253,22 @@ pipeline {
                         }
                         aborted {
                             script {
-                                def stackName = "test-single-${BUILD_ID}-${ARCHITECTURE}-perf-test"
+                                stackName = "test-single-${BUILD_ID}-${ARCHITECTURE}-${BUILD_NUMBER}"
                                 withCredentials([string(credentialsId: 'jenkins-aws-account-public', variable: 'AWS_ACCOUNT_PUBLIC')]) {
-                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region:'eu-west-1') {
```

Comment:
nit: 
```suggestion
                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region: 'eu-west-1') {
```

Reply:
oops // sorry about that .. let me correct it !! thanks !!